scripts/bump.sh

```diff
@@ -2,19 +2,37 @@
 
 set -eu
 
+# For local development, in production, the environment will be set though GH actions and GH secrets
+if [ -f ".envrc" ]; then
+    echo "Loading .envrc"
+    # shellcheck disable=SC1091
+    . .envrc
+else
+    echo "No .envrc"
+fi
+
+
 RELEASE_CHANNEL=$1
 echo "RELEASE_CHANNEL: $RELEASE_CHANNEL"
 
-NPM_VERSION=$2
-echo "NPM_VERSION: $NPM_VERSION"
+PRISMA_VERSION=$(sh scripts/prisma-version.sh "$RELEASE_CHANNEL")
+echo "UPDATING to $PRISMA_VERSION"
+echo "PRISMA_VERSION: $PRISMA_VERSION"
+
 
 PRISMA_CHANNEL=$RELEASE_CHANNEL
 echo "PRISMA_CHANNEL=$PRISMA_CHANNEL"
 
 OLD_SHA=$(jq ".prisma.version" ./package.json)
 SHA=$(npx -q -p @prisma/cli@"$PRISMA_CHANNEL" prisma --version | grep "Query Engine" | awk '{print $5}')
 
-NEXT_EXTENSION_VERSION=$3
+NPM_VERSION=$(sh scripts/prisma-version.sh "$RELEASE_CHANNEL")
```

Comment:
Line numbers 18.29 look like duplicates and can be consolidated into one variable

![image](https://user-images.githubusercontent.com/746482/84917091-82161380-b0dc-11ea-9605-ee324531f279.png)

Reply:
Other than this, I tested the file manually and it seems to be doing what it is supposed to do.